### PR TITLE
solved issue  #9832

### PR DESCRIPTION
**Pull request by yashkhandelwal123**
## One-line summary

This changeset (explain what) in order to (why).

## Significant changes and points to review

(Outline the main things changed, and if there are particular parts that need extra scrutiny.)

## Screenshots

(If the changeset updates the UI, please include screenshots so reviewers know what to look for when testing)

## Checklist

If relevant:

- [ ] Tests added
- [ ] Feature flags added to www-config
- [ ] New secrets added to the secrets repository
- [ ] If new dependencies are added, I've checked their license is appropriate

## Testing

Demo server URL: (or None)

To test this work:

- [ ]

### CODE REVIEW

**Review by alexgibson — CHANGES_REQUESTED** [review]
Thanks @yashkhandelwal123!

Can we please add this as a new string instead?

File: l10n/en/firefox/channel.ftl [hunk 1]
```diff
@@ -23,7 +23,7 @@ firefox-channel-all-languages-and-builds = All Languages and Builds
 
 # Variables:
 #   $link (url) - link to https://www.mozilla.org/privacy/firefox/#pre-release
-firefox-channel-beta-is-an-unstable-testing = { -brand-name-beta } is an unstable testing and development platform. By default, { -brand-name-beta } sends data to { -brand-name-mozilla } — and sometimes our partners — to help us handle problems and try ideas. <a href="{ $link }">Learn what is shared</a>.
+firefox-channel-beta-is-an-unstable-testing = { -brand-name-beta } is an unstable testing and development platform. By default, { -brand-name-beta } sends data to { -brand-name-mozilla } — and sometimes our partners — to help us handle problems and try ideas. <a href="{ $link }">Learn what is shared</a>.<br><p>NOTE :- The app will be updated approximately once a day.</p>
```
[the remark this answers is not in the document]
Comment (yashkhandelwal123) [reply l10n/en/firefox/channel.ftl:26]: Thank you for your message @alexgibson .I will add the provided information as a new paragraph, separate from the existing content.
If you have any specific details or content that you would like me to include, kindly provide them.

[the remark this answers is not in the document]
Comment (alexgibson) [reply l10n/en/firefox/channel.ftl:26]: Let's go with "Note: Firefox Nightly will update at least once per day."

File: bedrock/firefox/templates/firefox/channel/android.html [hunk 2]
```diff
@@ -45,6 +45,9 @@
         <a href="{{ url('privacy.notices.firefox') + '#pre-release' }}">{{ ftl('ui-learn-more') }}</a>
       {% endif %}
     </p>
+    <p>
+      {{ firefox-channel-beta-update-one-or-more-times }}
+    </p>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/channel/android.html:50]: This should be in the Firefox Nightly section of the page, not the Firefox Beta section. 

It also throws an error as the syntax is incorrect. Please use the following:

```
{% if ftl_has_messages('firefox-channel-beta-update-one-or-more-times') %}
  <p>
    {{ ftl('firefox-channel-beta-update-one-or-more-times') }}
  </p>
{% endif %}
```

File: bedrock/firefox/templates/firefox/channel/desktop.html [hunk 3]
```diff
@@ -43,6 +43,9 @@
         <a href="{{ url('privacy.notices.firefox') + '#pre-release' }}">{{ ftl('ui-learn-more') }}</a>
       {% endif %}
     </p>
+    <p>
+      {{ firefox-channel-beta-update-one-or-more-times }}
+    </p>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/channel/desktop.html:48]: Same issue as above: in the wrong place, and also throws an error.

**Review by alexgibson — CHANGES_REQUESTED** [review]
Thanks for the update @yashkhandelwal123 - some fixes below.

Please test your changes before submitting them for review. Thanks!

**Comment by alexgibson** [comment]
Closing due to inactivity. If you find time to revisit this, feel free to reopen!

**Pull request closed without merging**